docs/source/daml2ts/index.rst

```diff
@@ -0,0 +1,291 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Typescript!
+###########
+
+daml2ts : Typescript of DAML
+============================
+
+Introduction
+------------
+``daml2ts`` generates `json-api <../json-api/index.html>`_ compatible Typescript code. It accepts compiled DAML modules as input. The Typescript source files it produces contain type, value and function definitions that model the DAML entities it finds.
+
+The code that ``daml2ts`` generates is expressed in terms of the packages `@digital-asset/daml-types <https://github.com/digital-asset/daml/tree/master/language-support/ts/daml-types>`_ and `@mojotech/json-type-validation <https://github.com/mojotech/json-type-validation>`_ packages. Accordingly, every Typescript file generated begins with the following snippet.
+
+.. code-block:: typescript
+   :linenos:
+
+   /* eslint-disable @typescript-eslint/camelcase */
+   /* eslint-disable @typescript-eslint/no-use-before-define */
+   import * as jtv from '@mojotech/json-type-validation';
+   import * as daml from '@digitalasset/daml-types';
```

Comment:
I don't think showing this preamble is very helpful. The lint warnings are not relevant to the user, `json-type-validation` is an implementation detail and the import of `@daml/types` follows from what's written above.

Reply:
✔️